src/components/ModalWindows/UserCabinetModalWindows/ChangeEmailModal.vue

```diff
@@ -96,126 +100,98 @@ export default {
     },
   },
   emits: ['closeModal', 'email'],
-  setup(props, context) {
-    const { t } = useI18n()
-    const modalSteps = ref({
-      first: true,
-      second: false,
-    })
-    const repeatEmail = ref('')
-    const errorMessage = ref('')
-
-    const schema = computed(() => {
-      return yup.object({
-        verify_code: yup
-          .string()
-          .verifyCodeRequired('errors.required')
-          .verifyCodeMinLength('errors.min5'),
-        email: yup.string().required('errors.required').email('errors.email'),
-      })
-    })
-
-    yup.addMethod(yup.string, "verifyCodeRequired", function (errorMessage) {
-      return this.test(`test-verify-code-required`, errorMessage, function (value) {
-        const { path, createError } = this;
-      
-        return (
-          (modalSteps.value.second && !value  ? false : true) || createError({ path, errorMessage })
-        );
-      });
-    });
-
-    yup.addMethod(yup.string, "verifyCodeMinLength", function (errorMessage) {
-      return this.test(`test-verify-code-min-length`, errorMessage, function (value) {
-        const { path, createError } = this;
-        return (
-          (modalSteps.value.second && value.length !== 5  ? false : true) || createError({ path, errorMessage })
-        );
-      });
-    });
+  setup(props, { emit }) {
+    const { t } = useI18n();
+    const loading = ref(false);
+    const toast = useToast();
+    const currentStep = ref(1);
 
     const cancelBtnTitle = computed(() => {
-      return modalSteps.value.first
+      return currentStep.value === 1
```

Comment:
will be great to prepare variable with type enum for steps 

const sterEnum = {
   firstStep: 1
   
   }
   
   
   
   return currentStep.value === sterEnum.firstStep